<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentsManager</class>
 <widget class="QDialog" name="StudentsManager">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire des élèves</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nom : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_name"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Prénom : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_firstName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Email :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_email"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="list_students"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_add">
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_update">
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_delete">
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_import">
         <property name="text">
          <string>Importer</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_close">
         <property name="text">
          <string>Fermer</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>StudentsManager</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>825</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
